kernel/arch/dreamcast/hardware/modem/mintr.c

```diff
@@ -12,50 +13,48 @@
 #include <dc/asic.h>
 #include <arch/rtc.h>
 #include <kos.h>
+#include <kos/oneshot_timer.h>
```

Comment:
I take it this means we are intentionally not adding `oneshot_timer.h` to `kos.h`?

Reply:
That `#include <kos.h>` shouldn't be there at all. Wasn't meant for internal use. This file and modem.c seem to be the only ones in the codebase that do this. Should get yanked and whatever other internals needed added instead with a smaller scope.

Reply:
Yes. But to be fair that's a bit outside the scope of this PR ;)

Reply:
Yeah, I guess all of the modem code could use an independent cleanup PR... Fair. lol.

Reply:
I can knock that out in a follow PR real quick as soon as this one gets merged

Reply:
Follow up here: https://github.com/KallistiOS/KallistiOS/pull/593